types.go

```diff
@@ -231,6 +231,22 @@ type OpenOrder struct {
 	Timestamp int64   `json:"timestamp"`
 }
 
+type FrontendOpenOrder struct {
```

Comment:
This type seems to overlap with QueriedOrder at https://github.com/sonirico/go-hyperliquid/pull/62/files#diff-b990161986c40e0867e71c60779bc98a8730a9eb0ca5866ae8189e126863c059R257. Which one do you suggest to keep?